exercises/concept/lasagna-master/lasagna-master.spec.js

```diff
@@ -0,0 +1,214 @@
+import {
+  cookingStatus,
+  preparationTime,
+  quantities,
+  addSecretIngredient,
+  scaleRecipe,
+} from './lasagna-master';
+
+const DIFFERENCE_PRECISION_IN_DIGITS = 6;
+
+describe('cookingStatus', () => {
+  test('recognizes that there is time left on the timer', () => {
+    const expected = 'Not done, please wait.';
+    expect(cookingStatus(1)).toBe(expected);
+    expect(cookingStatus(42)).toBe(expected);
+    expect(cookingStatus(8.5)).toBe(expected);
+    expect(cookingStatus(0.1)).toBe(expected);
+  });
+
+  test('recognizes when there is no time left on the timer', () => {
+    expect(cookingStatus(0)).toBe('Lasagna is done.');
```

Comment:
This made me think: negative numbers are undefined because it's a timer. Perhaps it makes sense to put in the task: "The timer can never go below 0".

Reply:
Added to the instructions.